python-client/pyproject.toml

```diff
@@ -159,9 +160,11 @@ llm = [
     "torch",
     "langchain",
     "evaluate",
-#    pdm lock -G:all doesn't work without fixing these two versions
+    #    pdm lock -G:all doesn't work without fixing these two versions
     "datasets>=2.13.0",
     "bert-score>=0.3.13",
+    "detoxify",
```

Comment:
Exact, but why here there is no version string?

Reply:
DONE